config/webpack/webpack.dev.config.js

```diff
@@ -0,0 +1,43 @@
+var path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './index.ts',
+  output: {
+    path: path.resolve(__dirname, '..', '..', 'dist', 'bundles'),
+    filename: 'help-client.umd.js',
+    library: 'BBHelpClient',
+    libraryTarget: 'umd'
+  },
+  devtool: 'inline-source-map',
+  resolve: {
+    extensions: ['.ts', '.js']
+  },
+  module: {
+    rules: [
+      {
+        test: /\.ts$/,
+        loader: 'ts-loader'
+      },
+      {
+        test: /\.scss$/,
+        loaders: ['style-loader', 'css-loader?localIdentName=[hash:base64:10]', 'sass-loader'],
```

Comment:
Curious about the `css-loader` being included, since the `test` only looks for SCSS files?

Reply:
css-loader is included because of how webpack runs its loaders.   Right to left.  So it takes the scss files, compiles them to css with the `sass-loader` it then runs it through the `css-loader` for common js, and then into the `style-loader` to create style nodes from the strings.  It's complicated but necessary. https://github.com/webpack-contrib/sass-loader

Reply:
Gotcha; so as long as you don't wish to support standard CSS, this is great!